subsystem-arm/implementations/hand/hand-modes.hpp

```diff
@@ -1,47 +1,53 @@
 #pragma once
-#include "dto/hand-dto.hpp"
 #include "peripherals/lpc40xx/i2c.hpp"
-#include "pca9685.hpp"
+#include "../implementations/pca9685.hpp" 
+#include "hand-router.hpp"
 
 namespace sjsu::arm
 {
     class HandModes
     {
     public:
 
-        static hand_arguments CloseHand(hand_arguments arguments_)
+        static hand_arguments CloseMode(hand_arguments arguments)
         {
-            arguments_.pinky_angle = min_angle_;
-            arguments_.ring_angle = min_angle_;
-            arguments_.middle_angle = min_angle_;
-            arguments_.index_angle = min_angle_;
-            arguments_.thumb_angle = min_angle_;
-            return arguments_;
+            arguments.pinky_angle = min_angle_;
+            arguments.ring_angle = min_angle_;
+            arguments.middle_angle = min_angle_;
+            arguments.index_angle = min_angle_;
+            arguments.thumb_angle = min_angle_;
+            return arguments;
         }
 
-        static hand_arguments OpenHand(hand_arguments arguments_)
+        static hand_arguments OpenMode(hand_arguments arguments)
         {
-            arguments_.pinky_angle = max_angle_;
-            arguments_.ring_angle = max_angle_;
-            arguments_.middle_angle = max_angle_;
-            arguments_.index_angle = max_angle_;
-            arguments_.thumb_angle = max_angle_;
-            return arguments_;
+            arguments.pinky_angle = max_angle_;
+            arguments.ring_angle = max_angle_;
+            arguments.middle_angle = max_angle_;
+            arguments.index_angle = max_angle_;
+            arguments.thumb_angle = max_angle_;
+            return arguments;
         }
 
-        static hand_arguments PointHand(hand_arguments arguments_)
+        static hand_arguments ManualMode(hand_arguments arguments)
         {
-            arguments_.pinky_angle = min_angle_;
-            arguments_.ring_angle = min_angle_;
-            arguments_.middle_angle = min_angle_;
-            arguments_.index_angle = neutral_angle_;
-            arguments_.thumb_angle = max_angle_;
-            return arguments_;
+            return arguments;
         }
 
+        static hand_arguments ClawMode(hand_arguments arguments)
+        {
+
+        }
```

Comment:
defeats the purpose of 5 finger hand.

Reply:
Can you explain more about why this mode defeats the purpose of 5 finger hands?